src/routes/ChatRoute/ChatRoute.jsx

```diff
@@ -8,9 +8,9 @@ export default function ChatPage() {
   return (
     <Container>
       <SearchBar />
-      <ChatWrapper>
+      <ChatStyled>
```

Comment:
ChatStyled should be moved inside Chat component.

Reply:
Changed.
And in the next commit I also changed the functions in the routes to arrow functions.